tests/integration/kubernetes/tests_common.sh

```diff
@@ -147,6 +147,10 @@ adapt_common_policy_settings() {
 	case "${KATA_HYPERVISOR}" in
   		"qemu-sev")
 			adapt_common_policy_settings_for_sev "${settings_dir}"
+			;;
+		"qemu-snp")
```

Comment:
You can merge it in the previous qemu-sev block, like:

```
qemu-sev|qemu-snp)
   adapt_common_policy_settings_for_sev "${settings_dir}"
...
...
```

Reply:
Updated. Thanks @wainersm  !